src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -14,19 +43,36 @@ export class ItemLogViewComponent implements OnChanges {
 
   @Input() itemData?: ItemData;
 
+  @ViewChild('table') private table?: Table;
+
   private readonly id$ = new ReplaySubject<string>(1);
   readonly state$ = this.id$.pipe(
     distinct(),
-    switchMap(id => this.activityLogService.getActivityLog(id)),
+    switchMap(id => this.getData$(id)),
     mapToFetchState(),
   );
 
   constructor(
-    private activityLogService: ActivityLogService
+    private activityLogService: ActivityLogService,
+    private router: Router,
   ) {}
 
   ngOnChanges(): void {
     if (this.itemData) this.id$.next(this.itemData.item.id);
   }
 
+  getData$(id: string): Observable<Data> {
+    return this.activityLogService.getActivityLog(id).pipe(
+      map((data: ActivityLog[]) => ({
+        columns: logColumns,
+        rowData: data
+      }))
+    );
+  }
+
+  navigateToItem(log: ActivityLog): void {
+    const itemTypeCategory = typeCategoryOfItem(log.item);
+    void this.router.navigateByUrl(incompleteItemStringUrl(log.item.id, itemTypeCategory));
+  }
```

Comment:
As it is a "basic" static link with no special action, I would have added it in the data directly.
Actually, we could create a custom pipe to map an item with `{id:string, type:string}` to a link directly in the template, it would be cleaner. If you don't get what I mean, leave it as it is and I'll change this later.

Reply:
fixed